test/Common.cpp

```diff
@@ -207,6 +207,38 @@ bool CommonOptions::parse(int argc, char const* const* argv)
 	return true;
 }
 
+std::string CommonOptions::toString(const std::vector<std::string>& _filter) const
+{
+	std::stringstream options;
+	if(!m_singleton || _filter.empty())
+		return "";
+
+	//Using std::map to avoid if-else/switch-case block
+	std::map<std::string, std::string> option_value_str = {
+		{"evmVersion", "evmVersion=" + evmVersion().name()},
+		{"optimize", "optimize=" + boolToString(optimize)},
+		{"useABIEncoderV1", "useABIEncoderV1=" + boolToString(useABIEncoderV1)},
+		{"batch", "batch=" + std::to_string(selectedBatch+1) + "/" + std::to_string(batches)},
+		{"ewasm", "ewasm=" + boolToString(ewasm)},
+		{"enforceCompileToEwasm", "enforceCompileToEwasm=" + boolToString(enforceCompileToEwasm)},
+		{"enforceGasTest", "enforceGasTest=" + boolToString(enforceGasTest)},
+		{"enforceGasTestMinValue", "enforceGasTestMinValue=" + enforceGasTestMinValue.str()},
+		{"disableSemanticTests", "disableSemanticTests=" + boolToString(disableSemanticTests)},
+		{"disableSMT", "disableSMT=" + boolToString(disableSMT)},
+		{"showMessages", "showMessages=" + boolToString(showMessages)},
+		{"showMetadata", "showMetadata=" + boolToString(showMetadata)}
+	};
+
+	for(auto optionName : _filter)
```

Comment:
Our style is to always but a space betwene `if` and `(` or `for` and `(`.
```suggestion
	for (auto optionName: _filter)
```